src/util-classification-config.c

```diff
@@ -253,8 +253,10 @@ int SCClassConfAddClasstype(DetectEngineCtx *de_ctx, char *rawstr, uint16_t inde
 
     ret = pcre2_match(regex, (PCRE2_SPTR8)rawstr, strlen(rawstr), 0, 0, regex_match, NULL);
     if (ret < 0) {
-        SCLogError(SC_ERR_INVALID_SIGNATURE, "Invalid Classtype in "
-                   "classification.config file");
+        SCLogError(SC_ERR_INVALID_SIGNATURE,
+                "Invalid Classtype in "
```

Comment:
would expect `Invalid Classtype '%s' in classification.config file`. I first read the error as "we found an error in file X".

Maybe even better would be `Invalid Classtype '%s' in classification.config file %s`